g2pf/client.py

```diff
@@ -22,3 +48,13 @@ def federated_featurephenotypeassociaton_query(self, args):
         """Search all clients for feature-phenotype associations as defined by
         args."""
         pass
+
+
+#Code for trial
```

Comment:
I created a test suite framework that you can modify. Go ahead and merge [the branch](https://github.com/ahwagner/g2p-federated/tree/ozanozisik-master) into your PR, and then modify [the tests](https://github.com/ahwagner/g2p-federated/blob/ozanozisik-master/tests/test_clientManager.py#L5-L9) accordingly.